src/main/assembly/dist/install/db-tables.sql

```diff
@@ -1,5 +1,16 @@
-CREATE TABLE seed (
-    type VARCHAR(64) NOT NULL UNIQUE,
-    value BIGINT NOT NULL);
+CREATE TABLE seed(
+    type VARCHAR(64) NOT NULL,
+    value BIGINT NOT NULL,
+    PRIMARY KEY (type)
+);
 
-GRANT INSERT, SELECT, UPDATE, DELETE ON seed TO easy_pid_generator;
+CREATE TABLE minted(
+    type VARCHAR(64) NOT NULL,
+    value VARCHAR(64) NOT NULL,
+    created TIMESTAMP NOT NULL,
```

Comment:
this change to `TIMESTAMP` doesn't work in PostgreSQL, I get an error. Need to look into that on monday.

```
ERROR The database connection failed; cause: ERROR: column "created" is of type timestamp without time zone but expression is of type character varying
  Hint: You will need to rewrite or cast the expression.
  Position: 59
```

Reply:
Solved, needs to be `TIMESTAMP WITH TIME ZONE` and this needs to be reflected by the calls in `PreparedStatement` and `ResultSet`